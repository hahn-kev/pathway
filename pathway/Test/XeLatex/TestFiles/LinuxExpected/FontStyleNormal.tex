\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\normal="Charis SIL" at 24pt
\font\italicnormal="Charis SIL/I" at 24pt
\font\normalitalicnormal="Charis SIL" at 24pt

\newpage 
\thispagestyle{empty} 
\mbox{} 
\newpage 
\newpage 
\setcounter{page}{1} 
\pagenumbering{arabic} 
\pagestyle{fancy} 

\normal{To test the font style }\italicnormal{(italic coming inside) }\normalitalicnormal{with normal }\normal{text }

\end{document}
