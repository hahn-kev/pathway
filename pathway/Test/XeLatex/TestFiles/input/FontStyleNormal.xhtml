<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html lang="utf-8" xmlns="http://www.w3.org/1999/xhtml" xml:lang="utf-8">
    <head>
        <title></title>
        <link rel="stylesheet" href="FontStyleNormal.css" type="text/css"/>
    </head>
    <body>
        <div class="normal">
            To test the font style 
            <span class="italic"> (italic coming inside) <span class="normal"> with normal </span></span> 
            text
        </div>
    </body>
</html>