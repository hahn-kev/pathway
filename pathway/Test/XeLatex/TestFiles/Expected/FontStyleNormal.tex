\documentclass[a4paper]{article} 
\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage[margin=2cm,includeheadfoot]{geometry}

\begin{document} 
\pagestyle{plain} 
\font\normalitalicnormal="Times New Roman" at 24pt
\font\italicnormal="Times New Roman/I" at 24pt
\font\normal="Times New Roman" at 24pt
\pagestyle{fancy} 

\normal{To test the font style }\italicnormal{(italic coming inside) }\normalitalicnormal{with normal }\normal{text }

\end{document}
